bigchaindb/common/transaction.py

```diff
@@ -421,7 +420,7 @@ class Transaction(object):
     TRANSFER = 'TRANSFER'
     GENESIS = 'GENESIS'
     ALLOWED_OPERATIONS = (CREATE, TRANSFER, GENESIS)
-    VERSION = '.'.join(bigchaindb.version.__short_version__.split('.')[:2])
+    VERSION = '1.0'
```

Comment:
Transaction version is breaking step with server version, so it makes sense to hardcode it in the code that generates the transactions. When the server version changes, the transaction version should not neccesarily change.